README.md

```diff
@@ -203,8 +209,10 @@ jobs:
       - name: pr-custom-review
         uses: paritytech/pr-custom-review@tag           # Pick a release tag and put it after the "@".
         with:
-          # A token with read-only organization permission is required for
-          # requesting reviews from teams.
+          # The token needs the following permissions
+          # - `read:org` for being able to request reviews from teams
+          # - `workflow` for being able to request the workflow's job
+          #    information; used to track lines in the job's output
```

Comment:
Don't you think this (or at least the required scopes) should be mentioned in the p2 of the Deployment chapter?

Reply:
8ae78bede894f3a5400398746af41871fa8e57ff